include/grpcpp/impl/codegen/call.h

```diff
@@ -654,7 +657,12 @@ class CallOpSet : public CallOpSetInterface,
 
   void set_output_tag(void* return_tag) { return_tag_ = return_tag; }
 
+  void* cq_tag() override { return cq_tag_; }
+
+  void set_cq_tag(void* cq_tag) { cq_tag_ = cq_tag; }
```

Comment:
If I understand correctly, setting a cq_tag other than this means we are by-passing the FinalizeResult of this class, and thus it needs to be called explicitly in the new tag. We should document this.

Reply:
I've documented this better now. "set_cq_tag" changes the tag that is passed down to core. The story with FinalizeResult is different, and I'll document that as well. In the sync or existing async API, FinalizeResult is called in Pluck or Next. In the callback-based API, FinalizeResult must be called by the wrapper of the callback function (since there is no Pluck or Next). So we're not bypassing FinalizeResult; we're bypassing Next and as a result have to call FinalizeResult explicitly.

Reply:
I will resolve this by improving the comment of FinalizeResult